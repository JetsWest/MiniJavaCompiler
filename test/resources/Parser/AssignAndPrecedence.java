class A {
    public static void main(String[] args) {
        System.out.println(new B().And());
    }

}
class B {
    public boolean And() {
        boolean a;
        boolean b;
        boolean c;
        a = false;
        b = true;
        c = a && b;
        return c;
    }
}