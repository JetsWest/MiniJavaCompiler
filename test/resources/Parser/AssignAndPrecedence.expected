Program
  MainClass A
    Print
      new B().And()
  Class B
    MethodDecl And
      returns boolean
      boolean a
      boolean b
      boolean c
      a = false
      b = true
      c = (a && b)
      Return c